scripts/sync_codemirror.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+# vim: ts=2 et
+# Setting -x is absolutely forbidden as it could leak the GitHub token.
+set -uo pipefail
+
+# GITHUB_TOKEN required scope: repo.repo_public
+
+git_mail="prometheus-team@googlegroups.com"
+git_user="prombot"
+branch="repo_sync_codemirror"
+commit_msg="Update codemirror"
+pr_title="Synchronize codemirror from prometheus/prometheus"
+pr_msg="Propagating changes from prometheus/prometheus default branch."
+target_repo="prometheus-community/codemirror-promql"
+
+color_red='\e[31m'
+color_green='\e[32m'
+color_yellow='\e[33m'
+color_none='\e[0m'
+
+echo_red() {
+  echo -e "${color_red}$@${color_none}" 1>&2
+}
+
+echo_green() {
+  echo -e "${color_green}$@${color_none}" 1>&2
+}
+
+echo_yellow() {
+  echo -e "${color_yellow}$@${color_none}" 1>&2
+}
+
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
+if [ -z "${GITHUB_TOKEN}" ]; then
+  echo_red 'GitHub token (GITHUB_TOKEN) not set. Terminating.'
+  exit 1
+fi
+
+# List of files that should not be synced.
+excluded_files="CODE_OF_CONDUCT.md LICENSE Makefile.common SECURITY.md .yamllint MAINTAINERS.md"
```

Comment:
`CODE_OF_CONDUCT.md` and `LICENSE` should by sync right ? With the regular Apache License

Reply:
they will be synced by the previous script, independently.